keycast.el

```diff
@@ -62,7 +62,7 @@ into `mode-line-format' after the element specified here."
   "Whether enabling `keycast-mode' removes elements to the right.
 
 When this is non-nil, then enabling `keycast-mode' not only
-inserts `mode-line-keycast' into `mode-line-format' but also
+inserts `keycast-mode-line' into `mode-line-format' but also
```

Comment:
Fine by me, would do it like that nowadays too. I am under the impression that in recent years there was a shift from "prefix symbols with the package name, except..." to "always prefix all symbols with the package name".

Reply:
I've not come across any cases where an exception of the strict naming convention makes sense. Therefore I would go with the renaming here. We can also deprecate the old symbols for backward compatibility. So far I've not added deprecations.

Reply:
I though that putting `define-*` first made sense and if I remember correctly that also was necessary to automatically `defun`-like styling.

Reply:
> I though that putting define- first made sense and if I remember correctly that also was necessary to automatically get defun-like styling.

Ah right. That's an argument. But I think you can still get the styling by adjusting some variables. So in the greater scheme of things I prefer if the naming convention is followed strictly. Of course this is just my preference, but package-lint is also pushing me into that direction :)

Reply:
> We can also deprecate the old symbols for backward compatibility.

For `mode-line-keycast-format` that would probably make sense, but then the namespace pollution would stick around. Shouldn't be too hard to notice that something has changed and how to fix it, so I think it is fine to not define the aliases in this case.